infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -41,20 +45,41 @@
     volatile long serverEpoch;
 
     /** The {@link AbstractServer}s this {@link NettyServerRouter} routes messages for. */
-    final List<AbstractServer> servers;
+    private final ImmutableList<AbstractServer> servers;
+
+    /**
+     * The set of messages for which the layout and cluster id validations should be ignored.
+     */
+    private final Set<CorfuMsgType> ignoreLayoutMsgTypes = ImmutableSet.of(
```

Comment:
Can you put this in CorfuMsgType. IgnoreClusterId should be specified in similar way to ignoreEpoch

Reply:
Fixed